src/SocialServer/SocialServer.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Threading;
+using Melia.Shared;
+using Melia.Shared.Data.Database;
+using Melia.Shared.Network;
+using Melia.Social.Database;
+using Melia.Social.Network;
+using Yggdrasil.Logging;
+using Yggdrasil.Network.TCP;
+using Yggdrasil.Util;
+using Yggdrasil.Util.Commands;
+
+namespace Melia.Social
+{
+	/// <summary>
+	/// Represents a social server.
+	/// </summary>
+	public class SocialServer : Server
+	{
+		public readonly static SocialServer Instance = new SocialServer();
+
+		private TcpConnectionAcceptor<SocialConnection> _acceptor;
+
+		/// <summary>
+		/// Returns a reference to the server's packet handlers.
+		/// </summary>
+		public PacketHandler PacketHandler { get; } = new PacketHandler();
+
+		/// <summary>
+		/// Returns reference to the server's database interface.
+		/// </summary>
+		public SocialDb Database { get; } = new SocialDb();
+
+		private static long ChatId = 0x1FB0F00000000;
+		public long GetNewChatId()
```

Comment:
Doesn't it feel like a proper chat manager might be cleaner? This method could also use a comment about what exactly it does and returns.